test/test_clients_api.js

```diff
@@ -143,6 +161,14 @@ describe('Clients API', () => {
   });
 
   describe('GET ' + endpointPrefix + '/clients', () => {
+    beforeEach(done => {
+      db().then(models => {
+        models.Clients.destroy({
+          where: {}
+        }).then(() => { done() });
+      });
```

Comment:
nit: I realize you can do something simpler like:
```javascript
beforeEach(function() {
  return db().then(({ Clients }) => Clients.destroy({ where: {} }));
});
```

You can return promises from mocha's test functions :)